docs/configuration/overview/runtime.rst

```diff
@@ -33,7 +33,51 @@ override_subdirectory
   specified and the directory exists, configuration values within this directory will override those
   found in the primary subdirectory. This is useful when Envoy is deployed across many different
   types of servers. Sometimes it is useful to have a per service cluster directory for runtime
-  configuration.
+  configuration. See below for exactly how the override directory is used.
+
+File system layout
+------------------
+
+Various sections of the configuration guide describe the runtime settings that are available.
+For example, :ref:`here <config_cluster_manager_cluster_runtime>` are the runtime settings for
+upstream clusters.
+
+Assume the following configuration settings for runtime:
+
+* *symlink_root*: ``/srv/runtime/current``
+* *subdirectory*: ``envoy``
+* *override_subdirectory*: ```envoy_override``
+
+Each '.' in a runtime key indicates a new directory in the hierarchy, rooted at *symlink_root* +
+*subdirectory*. For example, the *health_check.min_interval* path would have the following full
+file system path:
+
+``/srv/runtime/current/envoy/health_check/min_interval``
+
+The terminal portion of the path is a file that generally contains a numeric value unless otherwise
```

Comment:
Break into smaller sentences? Like
The terminal portion of a path is the file. The contents of the file constitute the runtime value. When reading numeric values from a file, spaces and new lines will be ignored.